app/models/experiment.rb

```diff
@@ -635,16 +635,25 @@ def generate_parameter_values(parameter)
     when 'value'
       # checking parameters for alpha-numeric characters, '_', '-' and '.'
       if /\A((\w)|(-)|(\.))+\z/.match(parameter['value']).nil?
-        raise SecurityError.new("Insecure parameter given - #{parameter.to_s}")
+        if parameter['value'] == ''
+          raise SecurityError.new("Insecure empty parameter given for Single value")
```

Comment:
Czy mamy możliwość pobrania tutaj nazwy parametru? Wtedy byłoby lepiej:
Empty value given for <nazwa parametru>